src/arguments.cpp

```diff
@@ -419,6 +419,22 @@ Error Arguments::parse(const char* args) {
             CASE("reverse")
                 _reverse = true;
 
+            CASE("hb-file")
+                _heartbeat_file = value;
+
+            CASE("delayns")
+                _heartbeat_delay_ns = strtol(value, NULL, 0);
+
+            CASE("clock-unix") {
+                _heartbeat_unix_clock = true;
```

Comment:
We could directly store either `CLOCK_REALTIME` or `CLOCK_MONOTONIC` instead of two boolean flags here, and pass it directly to the constructor of `HeartBeatFilter` instead of having to call `clock_id(bool, bool)`